packages/deposit/example/main.dart

```diff
@@ -31,23 +31,32 @@ class ExampleAdapter extends DepositAdapter<int> {
   }
 
   @override
-  Future<bool> exists(String table, int id) async {
+  Future<bool> exists(String table, String primaryColumn, int id) async {
```

Comment:
What if we use named parameters and leave 'id' as a default?
```suggestion
  Future<bool> exists({ required String table, required int id, String primaryColumn = 'id' }) async {
```

Reply:
Well the adapter just gets it from the Deposit, which has it as a default parameter. So adding them as defaults on the adapters itself would be double I think? 

End users wont normally call the adapter code themselves and if they did, they probably want to define custom primaries anyway? So that is why I opted it for this

Reply:
Oh, yeah, that makes sense, this the adapter!